api_v3/services/LiveStreamService.php

```diff
@@ -760,4 +748,65 @@ protected function updateAdminTagsCounters(LiveEntry $entry, &$adminTagsCounters
 		return $counterChanged;
 	}
 
+	/**
+	 * Checking whether conversionProfileId is cloud transcode
+	 *
+	 * @param int $conversionProfileId
+	 * @return boolean
+	 */
+	protected function isCloudTranscode($conversionProfileId)
+	{
+		$flavorParamsConversionProfile = flavorParamsConversionProfilePeer::retrieveByConversionProfile($conversionProfileId);
+		foreach($flavorParamsConversionProfile as $flavorParamConversionProfile)
+		{
+			/* @var $flavorParamConversionProfile flavorParamsConversionProfile */
+			if($flavorParamConversionProfile->getOrigin() == KalturaAssetParamsOrigin::CONVERT)
+			{
+				return true;
+			}
+		}
+		return false;
+	}
+
+	/**
+	 * Checking whether $entry is limited by adminTags existing in $limitedAdminTags
+	 *
+	 * @param LiveEntry $entry
+	 * @param array $limitedAdminTags
+	 * @return boolean
+	 */
+	protected function isAdminTagLimited(LiveEntry $entry, $limitedAdminTags)
+	{
+		return count(array_intersect(explode(',', $entry->getAdminTags()), $limitedAdminTags)) !== 0;
+	}
+
+
+	/**
+	 * Validating adminTag limits not reached.
+	 *
+	 * @param LiveEntry $currentEntry
+	 * @param array $liveEntries
+	 * @param array $adminTagsOrigLimits
+	 * @throws KalturaAPIException
+	 */
+	protected function validateAdminTagLimits(LiveEntry $currentEntry, $liveEntries, $adminTagsOrigLimits)
+	{
+		KalturaLog::debug('Current AdminTags: [' . $currentEntry->getAdminTags() . '] AdminTag limits : [' . print_r($adminTagsOrigLimits, true) . ']');
```

Comment:
Better move this log to before the "if" that call this function - as if this function will not call but we think we should we never see the log